app/services/flow/flow_state_machine.rb

```diff
@@ -176,6 +176,9 @@ def analytics_properties
         step: current_step,
         step_count: current_flow_step_counts[current_step_name],
         analytics_id: @analytics_id,
+        irs_reproofing: effective_user&.decorate&.reproof_for_irs?(
+          service_provider: current_sp,
+        ).present?,
```

Comment:
Is `.present?` necessary? Would `reproof_from_irs?` ever return anything other than `true` or `false`?

```suggestion
        ),
```

Reply:
It returns nil also.

Reply:
Oh, `reproof_for_irs?` wouldn't, but I see now that with the safe navigation it might be possible that `effective_user` could.